cmd/edit/cluster/cmd.go

```diff
@@ -163,6 +172,52 @@ func init() {
 			"to be added to the Hosted Control Plane's VPC Endpoint Service to enable additional "+
 			"VPC Endpoint connection requests to be automatically accepted.",
 	)
+
+	flags.StringSliceVar(
+		&args.allowedRegistries,
+		"allowed-registries",
+		nil,
+		"A comma-separated list of registries for which image pull and push actions are allowed.",
+	)
+
+	flags.StringSliceVar(
+		&args.insecureRegistries,
+		"insecure-registries",
+		nil,
+		"A comma-separated list of registries which do not have a valid TLS certificate or only support HTTP connections.",
+	)
+
+	flags.StringSliceVar(
+		&args.blockedRegistries,
+		"blocked-registries",
+		nil,
+		"A comma-separated list of registries for which image pull and push actions are denied.",
+	)
+
+	flags.StringVar(
+		&args.allowedRegistriesForImport,
+		"allowed-registries-for-import",
+		"",
+		"Limits the container image registries from which normal users can import images. "+
+			"The format should be a comma-separated list of 'domanName:insecure'. "+
+			"'domainName' specifies a domain name for the registry. "+
+			"'insecure' indicates whether the registry is secure or insecure.",
+	)
+
+	flags.StringVar(
+		&args.platformAllowlist,
+		"platform-allowlist",
+		"",
+		"A comma-separated list of internal registries that needs to be whitelisted for the platform to work. "+
```

Comment:
this is a reference to a RegistryAllowlistId, not a list. 
I think we should change that comment + mark this as hidden for the initial phase, as we don't expect changes at the beginning, so no need to expose it to the user

Reply:
Hide the args and changed the text to "The id of the internal registry that needs to be whitelisted for the platform to work. It can be omitted at creation and updating and its lifecycle can be managed separately if needed."